driver-adapters-wasm/neon-cfpages-basic/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -eu
+
+pnpm install
+
+pnpm prisma generate
+
+pnpm wrangler pages deploy . --project-name neon-cfpages-basic | tee deployment-logs.txt
+sleep 5
```

Comment:
Why do we need to wait a bit?

Reply:
Because otherwise the deployments often do not exist yet and you get a Cloudflare error message :/